### PR TITLE
Fixed #33277 -- Blocked database connections in threads for SimpleTestCase.

### PR DESCRIPTION
**Pull request by David-Wobrock**
[ticket-33277](https://code.djangoproject.com/ticket/33277)

### CODE REVIEW

**Comment by felixxm** [comment]
@David-Wobrock Do you have time to keep working on this?

**Comment by David-Wobrock** [comment]
Hey @felixxm 
I haven't found the time yet to finish the work 😕 

I hope to be able to tackle it in the coming weeks, but if you want to pick it up from here, feel free 🙂

File: tests/test_utils/tests.py [hunk 1]
```diff
@@ -2029,6 +2053,19 @@ def test_allowed_database_queries(self):
     def test_allowed_database_chunked_cursor_queries(self):
         next(Car.objects.iterator(), None)
 
+    def test_allowed_threaded_database_queries(self):
+        car = Car.objects.create()
+
+        result = []
+
+        def thread_func():
+            result.append(Car.objects.first())
```
Comment (David-Wobrock) [inline tests/test_utils/tests.py:2062]: Fails with `sqlite3.OperationalError: no such table: test_utils_car` 🤨

**Comment by felixxm** [comment]
Closing due to inactivity.

**Pull request closed without merging**